node/src/chain_spec.rs

```diff
@@ -2,14 +2,14 @@ use node_template_runtime::{
     pallet_utxo, AccountId, AuraConfig, BalancesConfig, GenesisConfig, GrandpaConfig, PpConfig,
     Signature, SudoConfig, SystemConfig, UtxoConfig, WASM_BINARY,
 };
+use sc_network::config::MultiaddrWithPeerId;
 use sc_service::ChainType;
 use sp_consensus_aura::sr25519::AuthorityId as AuraId;
+use sp_core::H256;
```

Comment:
I got really confused as to why you were adding this for a second until I double checked and saw you removing it just below